aggregate_logging_sizing.adoc

```diff
@@ -0,0 +1,283 @@
+logging-fluentd-7dtom         0/1                      Pending       0                    2d
+logging-fluentd-8k847         1/1                      Running       0                    3d
+----
+
+You should end up with a similar setup to the below.
+
+----
+$ oc get pods -o wide
+
+NAME                          READY     STATUS      RESTARTS   AGE       NODE
+logging-curator-1-6s7wy       1/1       Running     0          3d        ip-172-31-24-239.us-west-2.compute.internal
+logging-deployer-un6ut        0/1       Completed   0          3d        ip-172-31-6-152.us-west-2.compute.internal
+logging-es-6cvk237t-1-cnpw3   1/1       Running     0          3d        ip-172-31-24-238.us-west-2.compute.internal
+logging-es-e5x4t4ai-1-v933h   1/1       Running     0          3d        ip-172-31-24-235.us-west-2.compute.internal
+logging-es-xmwvnorv-1-adr5x   1/1       Running     0          3d        ip-172-31-24-233.us-west-2.compute.internal
+logging-fluentd-156xn         1/1       Running     0          3d        ip-172-31-24-241.us-west-2.compute.internal
+logging-fluentd-40biz         1/1       Running     0          3d        ip-172-31-24-236.us-west-2.compute.internal
+logging-fluentd-7dtom         0/1       Pending     0          2d        ip-172-31-24-243.us-west-2.compute.internal
+logging-fluentd-8k847         1/1       Running     0          3d        ip-172-31-24-237.us-west-2.compute.internal
+logging-fluentd-9a3qx         1/1       Running     0          3d        ip-172-31-24-231.us-west-2.compute.internal
+logging-fluentd-abvgj         1/1       Running     0          3d        ip-172-31-24-228.us-west-2.compute.internal
+logging-fluentd-bh74n         1/1       Running     0          3d        ip-172-31-24-238.us-west-2.compute.internal
+...
+...
+
+----
+
+Notice how the pods are placed in different cluster nodes. +
+
+
+By default the amount of RAM allocated to each ES instance is 8GB. `ES_INSTANCE_RAM` is the parameter used in the openshift-ansible https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_examples/files/examples/v1.3/infrastructure-templates/enterprise/logging-deployer.yaml[template]. Keep in mind that *half* of this value will be passed to the individual elasticsearch pods java processes https://www.elastic.co/guide/en/elasticsearch/guide/current/heap-sizing.html#_give_half_your_memory_to_lucene[heap size].
+
+Further information regarding installing EFK can be found in the openshift-docs repository https://github.com/openshift/openshift-docs/blob/master/install_config/aggregate_logging.adoc[documentation]. 
+
+ 
+== Systemd-journald and rsyslog 
+
+*Rate-limiting*
+
+In RHEL7 the `systemd-journald.socket` unit creates `/dev/log` during the boot process, and then passes input to systemd-journald.service. Every `syslog()` call goes to the journal. +
+Rsyslog gets logs from the journal via the http://www.rsyslog.com/doc/v7-stable/configuration/modules/imjournal.html[imjournal module].
```

Comment:
we have official docs on this interaction here https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html-single/System_Administrators_Guide/index.html#s1-interaction_of_rsyslog_and_journal